nix/scripts.nix

```diff
@@ -1,18 +1,34 @@
 { pkgs
 , config
-}:
-rec {
+, cronos ? (import ../. { inherit pkgs; })
+, chainmain ? (import ../nix/chainmain.nix { inherit pkgs; })
+, hermes ? (import ../nix/hermes.nix { inherit pkgs; })
+
+}: rec {
+  start-chainmain = pkgs.writeShellScriptBin "start-chainmain" ''
+    export PATH=${pkgs.pystarport}/bin:${chainmain}/bin:$PATH
+    ${../scripts/start-chainmain} ${config.chainmain-config} $@
+  '';
   start-cronos = pkgs.writeShellScriptBin "start-cronos" ''
     # rely on environment to provide cronosd
     export PATH=${pkgs.pystarport}/bin:$PATH
     ${../scripts/start-cronos} ${config.cronos-config} $@
   '';
+  start-cronos2 = pkgs.writeShellScriptBin "start-cronos2" ''
+    export PATH=${pkgs.pystarport}/bin:${cronos}/bin:$PATH
+    ${../scripts/start-cronos} ${config.cronos-config} $@
```

Comment:
it's using the same configuration file as the `start-cronos`, why a different script?